openreview/venue/process/custom_stage_process.py

```diff
@@ -152,7 +155,7 @@ def process(client, edit, invitation):
             recipients=submission.content['authorids']['value'],
             ignoreRecipients=ignore_groups,
             replyTo=contact,
-            subject=f'''[{short_name}] A {invitation_name} has been received on your Paper Number: {submission.number}, Paper Title: "{submission.content['title']['value']}"''',
+            subject=f'''[{short_name}] {before_invitation} {invitation_name} has been received on your Paper Number: {submission.number}, Paper Title: "{submission.content['title']['value']}"''',
             message=f'''We have recieved a {invitation_name} on your submission to {short_name}
```

Comment:
```suggestion
            message=f'''We have received a {invitation_name} on your submission to {short_name}
```

Reply:
please include a test for the message content.